part0.md

```diff
@@ -0,0 +1,24 @@
+Percentile Exercises
+
+Part 0 - Statistics
+
+Read https://en.wikipedia.org/wiki/Percentile through the section on the  “The nearest-rank method” and https://en.wikipedia.org/wiki/Quantile through “Examples”
+
+0.1. Given 1000 measures: How many are above the 90th percentile?  How many are below the 90th percentile?  Above and below the 99.5th percentile?
+
+    - 900 are below, 100 are above
+    - 995 are below, 5 are above
+
+
+
+0.2 How many measures does a dataset need to have the 99.999th percentile be distinct from the 100th percentile (max) of a dataset?
+
+    - 100,000
+0.3 Given a large unknown dataset, what is the probability that one randomly selected measure exceeds the 99th percentile? What is the probability that at least one measure in a sample of 50 that exceeds the 99th percentile of the dataset?
+
+    - 0.5%
+    - 1%
```

Comment:
Both of these are incorrect.

A random measure should have a 1% chance of being over the 99th percentile. It'd need to be in the top 1%. Or `(1 - .99)` for probability.

For 50 random measures, we'd want to combine the chance that each one is not in the top 1%, or `.99 * .99 * .99 ...` aka `.99^50`.  However, that is the chance that they're all not in the 1%.  The chance that at least one is will be the  complement to this: `1 - .99^50`